src/Docker/Manager/ContainerManager.php

```diff
@@ -285,6 +285,28 @@ public function execstart($execid, callable $callback = null, $detach = false, $
     }
 
     /**
+     * Get details about an executive
+     *
+     * @param string $execid Identifier from exec()
+     *
+     * @throws \Docker\Exception\UnexpectedStatusCodeException
+     *
+     * @return stdClass
```

Comment:
In the case of `exec()` you might need/want to interact with the output as a stream or whatever, which is not the case here, or did I miss something?